src/Integration.Vsix/CFamily/ProcessRunner.cs

```diff
@@ -52,7 +54,7 @@ public ProcessRunner(ISonarLintSettings settings, ILogger logger)
         /// Runs the specified executable and returns a boolean indicating success or failure
         /// </summary>
         /// <remarks>The standard and error output will be streamed to the logger. Child processes do not inherit the env variables from the parent automatically</remarks>
-        public bool Execute(ProcessRunnerArguments runnerArgs)
+        public void Execute(ProcessRunnerArguments runnerArgs)
```

Comment:
I'm not sure about changing it to void.
We would no longer check for it anyway (except in tests), since we would handle the issues before the process ends.

Reply:
There's no point in having it if we don't use it, but I agree it seems odd. Maybe it's because it shows we don't provide any information about the success/failure of the execution.